code/modules/research/xenobiology/crossbreeding/_mobs.dm

```diff
@@ -43,3 +43,52 @@ Slimecrossing Mobs
 	speak_emote = list("blorbles", "bubbles", "borks")
 	emote_hear = list("bubbles!", "splorts.", "splops!")
 	emote_see = list("gets goop everywhere.", "flops.", "jiggles!")
+
+//Hostile Copy - Destabilized Cerulean
+/mob/living/simple_animal/hostile/clone
+	name = "hostile clone"
+	real_name = "hostile clone"
+	desc = "How familiar..."
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 0 //Human punch damage range
+	melee_damage_upper = 9
```

Comment:
if you wanted to go the extra mile, you could have it try disarm shoves in addition to harm intent punches

maybe you could turn this into a reskinned monkey (that can't bite people)? then you could give it a sword or something and have it use it.

Reply:
actually, since this works on simplemobs, maybe you could make this clone copy some of the cloned simplemob's stats (from non-megafauna) when it's created?

instead of doing the monkey thing, you could make it copy the force of the target's currently-held weapon (and use its fists if said force would be lower than 5 or so).

we already have an illusion mob for this kind of thing, by the way- it's used by the Grey Tide spear and I think maybe one or two other effects.

Reply:
The clones are purposefully weak to prevent the exact type of situation you are describing. The last thing we need is people creating their own battle summon with the same damage/health stats as a megafauna (or at least this is not the intended reach of this particular feature).

Reply:
The "clone" aspect of this extract should mostly be an aesthetic one, equivalent to that of a weak morph.